application/common/models/Mfa.php

```diff
@@ -400,4 +415,25 @@ public static function removeOldUnverifiedRecords($maxAgeHours = 2)
             'count' => $numDeleted,
         ]);
     }
+
+
+    protected function sendAppropriateMessages($user, $eventType)
```

Comment:
Both places we call this, we're using `self::sendAppropriateMessages(...)`, so it looks like it needs to be changed from `protected function` to `protected static function`.

Reply:
Done